docs/flow/reference/components/prometheus.scrape.md

```diff
@@ -140,15 +148,74 @@ components.
 
 ## Component health
 
-`prometheus.scrape` will only be reported as unhealthy when given an invalid
+`prometheus.scrape` is only reported as unhealthy if given an invalid
 configuration.
 
 ## Debug information
 
 `prometheus.scrape` reports the status of the last scrape for each configured
 scrape job on the component's debug endpoint.
 
-### Debug metrics
+## Debug metrics
 
 `prometheus.scrape` does not expose any component-specific debug metrics.
 
+## Scraping behavior
```

Comment:
I'm OK leaving this here for now, but I think after we have all the documentation fleshed out we'll want to spend a little more time making a conscious decision generally where detailed explanation sections should go in the layout of a component reference page.